### PR TITLE
chore: update README for online demo

### LINKED ISSUE #2369: Optimize demo access experience
# Feature request

## Please describe your feature

Just a UX optimization suggestion: in the README we have the address and account information for the demo

image

Since the whole is in the same block of code, when the user clicks copy he will get the whole block of information, which cannot be put into the browser to access. This can make things tedious.

## Describe the solution you'd like

1. Place the address and account information separately so that when the user clicks copy he can paste it directly into the browser to access.
2. Put a separate link above for example: 
We have an online demo, [click to visit](http://106.55.144.26/)

What do you think?

### PR DESCRIPTION
**Pull request by juxianzheng**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

resolve #2369

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by guoqqqi — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -46,8 +46,9 @@ Note: Currently the Dashboard does not have complete coverage of Apache APISIX f
 
 ## Demo
 
+[Online Playground](https://106.55.144.26/)
```
Comment (leslie-tsang) [inline README.md:49]:
```suggestion
[Online Playground](http://106.55.144.26/)
```

Seems this playground only support `http`

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (guoqqqi) [reply README.md:49]: Good catch!

Comment (juxianzheng) [reply README.md:49]: > 接得好！I changed https://106.55.144.26 / to http://106.55.144.26 / and submitted it again, did I receive a new PR?

**Review by SkyeYoung — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @juxianzheng  , thank you for your contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Comment by juxianzheng** [comment]
> 你好@juxianzheng ， 感谢您的贡献！
> 
> 这是贡献者 T 恤表格[1]，如果您有兴趣，请查看 :)
> 
> [1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt



> 你好@juxianzheng ，感谢您的贡献！
> 
> 这是作者 T 恤表格[1]，如果您有兴趣投稿，请查看 :)
> 
> [1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt



> 你好@juxianzheng ，感谢您的贡献！
> 
> 这是作者 T 恤表格[1]，如果您有兴趣投稿，请查看 :)
> 
> [1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

Okay, thanks.

**Comment by juxianzheng** [comment]
> Hi @juxianzheng , thank you for your contribution!
> 
> Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)
> 
> [1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

Okay, thanks.

**Pull request merged**